apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/R4CoverageResourceProviderE2E.java

```diff
@@ -75,57 +73,6 @@ public void readCoveragesForExistingBeneficiary() throws FHIRException {
     CoverageTransformerV2Test.assertPartDMatches(beneficiary, partDCoverage);
   }
 
-  /**
```

Comment:
Deleted tests here were moved into test cases in the unit tests for the resource provider.